bin/test-entrypoint.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+# Script stalls dev container until db comes up.
+
+RET=1
+echo "Waiting for database"
+while ! pg_isready -h postgres -d beluga -U beluga; do
```

Comment:
Is there no way to do this in the python app? An `onready` event or something in SQLAlchemy

Reply:
Not sure--I pulled this in @chadlagore branch. My gut feel is no based on the behavior I observed before pulling this script in, in which the Python app failed to connect over and over.

Reply:
Good call. The depends_on flag wasn’t quite handling it (it would wait till the container was up, but not until Postgres was accepting connections), but I bet there’s some way we can do it in before_server_start.